src/sources/available_screen_position.ts

```diff
@@ -0,0 +1,39 @@
+import { replaceNaN, round, toFloat } from '../utils/data'
+import * as browser from '../utils/browser'
+
+type Position = [left: number | null, top: number | null, width: number | null, height: number | null]
```

Comment:
this structure is usually called a Rect.
(e.g. Rect {x: int, y: int, width: int, height: int})
Example: https://docs.microsoft.com/en-us/dotnet/api/system.windows.rect?view=net-5.0

It's not really important, but I noticed that because it didn't really look like a position to me.

Reply:
`Rect` is "what it is" and `Position` is "what it means" (a higher level abstraction). I don't mind calling it `Rect`. But I think I'll remove this type soon to implement a better solution.